src/test/java/org/broadinstitute/hellbender/tools/walkers/CombineGVCFsIntegrationTest.java

```diff
@@ -365,4 +365,34 @@ public void testOneHasDeletionAndTwoHasRefBlock() throws Exception {
         Assert.assertEquals(third.getGenotypes().size(), 2);
     }
 
+    @Test
+    public void testStartChromosome() throws Exception {
+        final File output = createTempFile("genotypegvcf", ".vcf");
+
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        args.addReference(new File(b37_reference_20_21))
+                .addOutput(output);
+        args.addVCF(getTestFile("gvcfExample1.fullchrom.2021.vcf"));
+        args.addVCF(getTestFile("gvcfExample2.vcf"));
+        args.add("--" + CombineGVCFs.IGNORE_VARIANTS_THAT_START_OUTSIDE_INTERVAL );
```

Comment:
What's this for?  I don't see any provided intervals.

Reply:
In reality? Nothing it was just copied from above. I will remove it. The important piece of the test is that I'm checking a vcf file containing variants that start at position 1 of the chromosome, `CombineGVCFs.IGNORE_VARIANTS_THAT_START_OUTSIDE_INTERVAL` only applies in the case where there is an interval provided, I will remove it

Reply:
Though perhaps it should throw if there is no interval provided?